### ISSUE TITLE
TestSecondDeploymentFailedInitialRetries flaked

### ISSUE THREAD

**Issue by smarterclayton** [openshift/origin#1371]
```
I0318 21:58:16.604320    5569 recreate.go:74] Deployment config-2 successfully made active
--- FAIL: TestSecondDeploymentFailedInitialRetries (0.07s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4773dd]
goroutine 9 [running]:
testing.func·006()
	/usr/local/go/src/testing/testing.go:441 +0x249
github.com/openshift/origin/pkg/deploy/strategy/recreate.(*RecreateDeploymentStrategy).updateReplicas(0xc208067ec0, 0x0, 0x0, 0xc20818c098, 0x8, 0x1, 0x0, 0x0)
	github.com/openshift/origin/pkg/deploy/strategy/recreate/_test/_obj_test/recreate.go:89 +0x2dd
github.com/openshift/origin/pkg/deploy/strategy/recreate.(*RecreateDeploymentStrategy).Deploy(0xc208067ec0, 0xc20818a000, 0xc208189ef8, 0x1, 0x1, 0x0, 0x0)
	github.com/openshift/origin/pkg/deploy/strategy/recreate/_test/_obj_test/recreate.go:50 +0x395
github.com/openshift/origin/pkg/deploy/strategy/recreate.TestSecondDeploymentFailedInitialRetries(0xc208165d40)
	/home/travis/gopath/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/pkg/deploy/strategy/recreate/recreate_test.go:137 +0x5c5
testing.tRunner(0xc208165d40, 0x169c830)
	/usr/local/go/src/testing/testing.go:447 +0x134
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:555 +0xd4e
goroutine 1 [chan receive]:
testing.RunTests(0xb6ab48, 0x169c800, 0x4, 0x4, 0xc2080ae401)
	/usr/local/go/src/testing/testing.go:556 +0xdc3
testing.(*M).Run(0xc208046320, 0x16a9080)
	/usr/local/go/src/testing/testing.go:485 +0xe1
main.main()
	github.com/openshift/origin/pkg/deploy/strategy/recreate/_test/_testmain.go:104 +0x382
goroutine 5 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x16a0ca0)
	/home/travis/gopath/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x99
created by github.com/golang/glog.init·1
	/home/travis/gopath/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2ca
goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1
FAIL	github.com/openshift/origin/pkg/deploy/strategy/recreate	0.576s
```

**Comment by ironcladlou** [comment]
What branch was this run invoked from? I'm seeing a supicious line number discrepancy with master I'd like to rule out as related.

**Comment by smarterclayton** [comment]
https://travis-ci.org/openshift/origin/jobs/54909216

**Comment by ironcladlou** [comment]
It was right in front of me the whole time.

**Comment by ironcladlou** [comment]
Fixed by #1380.
